docs/src/pages/[platform]/components/radiogroupfield/examples/ThemeExample.tsx

```diff
@@ -1,33 +1,23 @@
-import {
-  Radio,
-  RadioGroupField,
-  ThemeProvider,
-  Theme,
-} from '@aws-amplify/ui-react';
+import { Radio, RadioGroupField, ThemeProvider } from '@aws-amplify/ui-react';
 
-const theme: Theme = {
-  name: 'radio-theme',
+const theme = {
+  name: 'radiogroup-theme',
   tokens: {
-    colors: {
-      border: {
-        // Radio focus state inherits this global focus color
-        focus: { value: '{colors.brand.secondary.80}' },
-      },
-      font: {
-        primary: { value: '{colors.brand.secondary.60}' },
-        secondary: { value: '{colors.brand.secondary.40}' },
-      },
-    },
     components: {
-      radio: {
+      radiogroup: {
+        label: {
+          color: { value: '{colors.brand.secondary.80}' },
+        },
         button: {
```

Comment:
Perhaps it'd be a bit clearer to call this `radio` rather than `button`?

Reply:
updated